tests/fixtures/plugwise/stretch_v31/notifications.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should this be an empty json object?

Reply:
Yes it should be empty, no notifications exist for the Stretch v3.
@CoMPaTech should there be 'empty' fixtures?

Reply:
Empty as in 'there is no notification' - IIRC there are others within our fixtures that are empty as well? (Only the ADAM one has one AFAIR)

Reply:
Yup: https://github.com/home-assistant/core/blob/dev/tests/fixtures/plugwise/anna_heatpump/notifications.json

Reply:
@MartinHjelmare its loaded in https://github.com/home-assistant/core/blob/0426b211f62a618f7277b9f178e832309699fbc5/tests/components/plugwise/conftest.py#L22

Reply:
I don't see any try/except in `tests/common.py` so I'd expect it to fail if the file doesn't exist.